safe/gui/widgets/dock.py

```diff
@@ -915,7 +914,7 @@ def layer_changed(self, layer):
             if show_keywords:
                 if inasafe_keyword_version_key not in keywords.keys():
                     show_keyword_version_message(
-                        self, 'No Version', self.inasafe_version)
```

Comment:
It's still possible if you have corrupted metadata xml file that doesn't have `keyword version` in it. I prefer to keep this althought the probability is very small.